feat(content): Hai Reveal: Add new mission A11-C to provide optional updates on the election. (#7771)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

mission "Hai Reveal [A11-C] Outgoing Election Update 1"
	landing
	source
		government "Hai"
		not planet "Hai-home"
		attributes "spaceport"
	to offer
		has "Hai Reveal [A11] Sound the Alarm: offered"
	on offer
		conversation

> 			`The spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be here strictly on business. The usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and goods with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of the non-essential sections of the spaceport have been closed.`

Reviewer: This is a landing mission, not a spaceport mission, so you are not actually in the spaceport during this mission.

Reviewer: I suggest you make this a spaceport mission, and give it the "major" trait, so no other spaceport missions are offered. Just make sure it doesn't trigger on Mountaintop. If there are several other missions like it, then the player's spaceport missions in Hai space are blocked until the election (A12) finishes, and they can keep getting election updates.

Reviewer: This apparently is one of *many* places in "hai reveal 1 intro.txt" where the player is not in the spaceport, but is told they're in the spaceport. I've opened an issue #7804 for that and noted that this PR adds another such bug.

Author: > This apparently is one of _many_ places in "hai reveal 1 intro.txt" where the player is not in the spaceport, but is told they're in the spaceport. 

A player must always pass *through* the spaceport to go anywhere. It's the port. You come in on a ship.
`spaceport` missions are for when the player deviates from whichever thoroughfare they must traverse to get from their ship to somewhere else in order to actually explore or spend time within the spaceport itself, rather than simply taking the most direct route through it.

But you *always* go through the spaceport. You've landed a ship, it's unavoidable by definition.

Reviewer: The player has deviated from the thoroughfare, to eat at a restaurant and ask questions of many Hai. That is the sort of thing you do when looking for spaceport missions. You are exploring and spending time within the spaceport itself. By your own definition, this is a spaceport mission.

Author: The food stalls have been moved into the thoroughfare because the rest of the spaceport is *closed*. That's established in the setup.

> 			`The spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be conducting strictly business affairs. The usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and good with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of non-essential sections of the spaceport have been closed.`

Reviewer 2: ```suggestion
			`This spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be conducting strictly business affairs; the usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and good with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of non-essential sections of the spaceport have been closed.`
```
A lot of similarly structured sentences is hard to read.

Author: @roadrunner56 this one doesn't sit well with me so I'd be keen for your opinion

> 			`The spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be conducting strictly business affairs. The usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and good with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of non-essential sections of the spaceport have been closed.`

Reviewer 3: What is `good with places to be.` supposed to mean?
```suggestion
			`The spaceport appears to be operating on a skeleton staff with an increased security presence. Plenty of ships are still coming and going, but they seem to be here strictly on business. The usual milling of tourists, haggling of merchants, and perusal of jobs all appear to have been suspended in favor of efficient boarding and disembarking of Hai with places to go and good with places to be. The displays in the main thoroughfare have been switched over to wall-to-wall election coverage and the rest of the non-essential sections of the spaceport have been closed.`
```

Reviewer 4: Likely a typo for 'those with places to be'?

Author: Should be *goods

As in people who have places to get to (for business), and goods with places to be (that they should be delivered to).

Reviewer 3: Updated accordingly.

> 			`	Some of the restaurants that are currently closed have set up temporary stands near the displays with the most standing room in the vicinity and are serving food to those who have no choice but to be here for some duration of time.`

Reviewer 2: ```suggestion
			`	Some of the restaurants that are currently closed have set up temporary stands near the displays with the most standing room in the vicinity, and are serving food to those who have no choice but to be here for some duration of time.`
```
Whoop whoop, it's the comma police. 
Explanation: "Some of the restaurants are serving food to those who have no choice but to be here" works, so a comma could go there as far as I understand it.

			choice
				`	(Get some food.)`
				`	(Ask around for an update.)`
					goto ask
				`	(Return to your ship.)`
					goto leave

> 			`	The stalls have a range of tasty Hai and Hai-human fusion cuisines in disposable takeaway trays. At first glance they appear to be some sort of cardboard equivalent, but a Hai near you finishes their meal and then begins nibbling on the tray afterwards. It would seem that they are edible.`

Author: ```suggestion
			`	The stalls have a range of tasty Hai and Hai-human fusion cuisines in disposable takeaway trays. At first glance they appear to be some sort of cardboard equivalent, but a Hai near you finishes their meal and then begins nibbling on the tray afterwards.`
```

			`	The election coverage is in Hai, naturally, but there's a strong sense that people are still scrambling to have relevant commentary on such short notice and with no prior opinion polling in lead-up. It certainly seems like things are heated. Different charts being presented for the same areas seem to suggest that there's a great deal of controversy and some very strong opinions being thrown around. A number of the Hai nearby make some sounds of concern occasionally, but they do not stay around to comment. Unfortunately, nothing comes up in your language by the time you've finished your food.`
			choice
				`	(Eat the tray.)`
					goto tray
				`	(Return to your ship.)`
					goto leave
			label tray

> 			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`

Author: Herby was indeed the intended term there, but otherwise yeah I think that structure is better.

> 			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`

Reviewer 3: It's up to you whether it's herby or herbal. I just didn't think herby was a word.

> 			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`

Author: Little of column a, little of b:
`	The tray is hard and difficult to break. It does have a remarkably intriguing combination of nutty and herby flavors that would probably be quite enjoyable, but you don't quite have the teeth for it.`

> 			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination nutty and herby flavor that would probably be quite enjoyable, but you don't have the teeth for it.`

Author: Little of column a, little of b:
```suggestion
			`	The tray is hard and difficult to break. It does have a remarkably intriguing combination of nutty and herby flavors that would probably be quite enjoyable, but you don't quite have the teeth for it.`
```

			choice
				`	(Ask around for an update.)`
					goto ask
				`	(Return to your ship.)`
					goto leave
			label ask

> 			`	When you ask around, a lot of Hai don't seem to be very interested in talking to you. One in particular looks like they recognized you and actively avoided you. After you've approached a half dozen Hai unsuccessfully, though, one takes pity and comes over towards you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`

Reviewer 5: ```suggestion
			`	When you ask around, a lot of Hai doesn't seem to be very interested in talking to you. One, in particular, looks like they recognized you and actively avoided you. After approaching a half dozen Hai unsuccessfully, one takes pity and comes to you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```

Reviewer 2: "A lot of Hai do not seem" is correct, "a lot of Hai does not seem" isn't.

Author: With Bene's fix
```suggestion
			`	When you ask around, a lot of Hai don't seem to be very interested in talking to you. One, in particular, looks like they recognized you and actively avoided you. After approaching a half dozen Hai unsuccessfully, one takes pity and comes to you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```

> 			`	When you ask around a lot of Hai don't seem to be very interested in talking to you. One in particular looks like they recognized you and actively avoided you. After you've approached a half dozen Hai unsuccessfully though, one takes pity and comes over towards you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`

Reviewer 2: ```suggestion
			`	When you ask around, a lot of Hai don't seem to be very interested in talking to you. One in particular looks like they recognized you and actively avoided you. After you've approached a half dozen Hai unsuccessfully, though, one takes pity and comes over towards you. He appears to be of middling years and extends a paw to shake your hand in a friendly manner.`
```

> 			`	"Apologies for my brethren's behavior Captain, this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare that they usually would."`

Reviewer 2: ```suggestion
			`	"Apologies for my brethren's behavior, Captain, this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare like they usually would."`
```

Reviewer 2: ```suggestion
			`	"Apologies for my brethrens' behavior, Captain, but this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare like they usually would."`
```

Reviewer 2: That's true. 
```suggestion
			`	"Apologies for my brethren's behavior, Captain, but this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare that they usually would."`
```
What about "that" --> "how"?

> 			`	"Apologies for my brethren's behavior, Captain, but this is a very important time for the Hai, and many of them are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare that they usually would."`

Reviewer 5: ```suggestion
			`	"Apologies for my brethren's behavior, Captain, but this is a very important time for the Hai. Many are under significant strain to be informed and also maintain their essential business as captains themselves. There are a number of regulations for Hai citizens to stay on top of if they are frequently moving between worlds during the period, and they haven't had the time to prepare then they usually would."`
```

			choice
				`	"Is politics a very important matter for the Hai?"`
					goto politics
				`	"What do they need to prepare for an election?"`

> 			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses' yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`

Reviewer 2: ```suggestion
			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have 'their fingers on the pulse', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```

Author: He's meant to get it wrong, address only grammar please.

Reviewer 2: It wasn't supposed to be that sort of change, I was even going to say that he should get it wrong, haha. 
```suggestion
			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```

> 			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds, and the subtleties of what makes them healthy, than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`

Reviewer 6: ```suggestion
			`	He gestures at the various Hai captains paying close attention to the election coverage or scrolling information feeds on their data pads. "No one is more closely tied to the specific dynamics of trade between our worlds and the subtleties of what makes them healthy than our merchant captains are. I believe the human phrase is that they have their, 'fingers on the pulses', yes? This election has nothing to do with economics, and yet all the candidates have historical economic platforms that they can be expected to bring with them. These captains are being bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more.`
```

This could go either way, depending on how you want the pacing of the speech to feel. If it were plain text instead of dialogue, those commas would be best as parentheses, though.

Reviewer 2: I suppose, but I do like breaks. Unless you get your diaphragm working you'll need to pause after "what makes them healthy", but I dunno how big Hai lungs are.

Reviewer 6: Not everyone moves away from the mic to breath in ;) A nice steady pace the whole way through the sentence works just fine.

It really comes down to if it's spoken straight through as a planned whole or if that phrase is being added in the moment as an additional thought.

Author: Grammatically speaking it's a nonessential adjectival clause where the "dynamics of trade" is the subject noun.
This means it should have commas, even if read straight through when spoken.

> 			`	"Ordinarily they would spend at least a month checking on different things and solidifying their positions before an election. A snap election has robbed them of that preparation time, and so they are stressed and concerned. Many, including myself, will have to put up with the consequences of this election for at least the next two decades before any meaningful correction can occur, should the outcome be unfortunate. Long is the timeline that a Hai merchant must consider when asked who should be supported in an election."`

Reviewer 2: ```suggestion
			`	"Ordinarily, they would spend at least a month checking on different things and solidifying their positions before an election. A snap election has robbed them of that preparation time, and so they are stressed and concerned. Many, including myself, will have to put up with the consequences of this election for at least the next two decades before any meaningful correction can occur, should the outcome be unfortunate. Long is the timeline that a Hai merchant must consider when asked who should be supported in an election."`
```
Probably an optional comma but I like to pause after words like "ordinarily".

				goto ping
			label politics

> 			`	He smiles. "You must not have considered the Hai lifespan Captain. A human might not understand the situation to have nuanced opinions until they are more than twenty years of age, and by the time they are fifty the outcome of their votes are unlikely to have a large effect on the remainder of their lives. It is comparatively easy, if things are okay, for you to simply ignore your politics in favor of other things. A Hai may live for over three hundred years, if they are quite fortunate, but will in most cases still be tying to make a livelihood halfway through that. Barring accident, stupidity, or malicious action, a Hai will typically live to see the longer term effects of their voting preferences. The more wealth a Hai has, the more significant those effects are likely to be. There are few Hai more politically aware than merchant captains, and fewer still whose opinions on the effects of change are trusted more widely.`

Reviewer 2: ```suggestion
			`	He smiles. "You must not have considered the Hai lifespan, Captain. A human might not understand the situation to have nuanced opinions until they are more than twenty years of age, and by the time they are fifty the outcome of their votes are unlikely to have a large effect on the remainder of their lives. It is comparatively easy, if things are okay, for you to simply ignore your politics in favor of other things. A Hai may live for over three hundred years, if they are quite fortunate, but will in most cases still be trying to make a livelihood halfway through that. Barring accident, stupidity, or malicious action, a Hai will typically live to see the longer term effects of their voting preferences. The more wealth a Hai has, the more significant those effects are likely to be. There are few Hai more politically aware than merchant captains, and fewer still whose opinions on the effects of change are trusted more widely.`
```

> 			`	He smiles. "You must not have considered the Hai lifespan, Captain. A human might not understand the situation to have nuanced opinions until they are more than twenty years of age, and by the time they are fifty the outcome of their votes are unlikely to have a large effect on the remainder of their lives. It is comparatively easy, if things are okay, for you to simply ignore your politics in favor of other things. A Hai may live for over three hundred years, if they are quite fortunate, but will in most cases still be trying to make a livelihood halfway through that. Barring accident, stupidity, or malicious action, a Hai will typically live to see the longer term effects of their voting preferences. The more wealth a Hai has, the more significant those effects are likely to be. There are few Hai more politically aware than merchant captains, and fewer still whose opinions on the effects of change are trusted more widely.`

Reviewer: I strongly question whether merchant captains are the most politically aware demographic, as they must spend most of their time being merchants and captains. There are many other occupations that involve investing money and researching politics. It is possible this individual is talking of his tail, though.

> 			`	"Our merchant captains are bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more. Ordinarily they would spend at least a month checking on different things and solidifying their positions before an election. A snap election has robbed them of that preparation time, and so they are stressed and concerned."`

Reviewer 2: ```suggestion
			`	"Our merchant captains are bombarded by queries from friends, families, organizations they trade with frequently, institutions they studied or supported, local media outlets, and more. Ordinarily, they would spend at least a month checking on different things and solidifying their positions before an election. A snap election has robbed them of that preparation time, and so they are stressed and concerned."`
```
It's optional, but it feels better.

			label ping

> 			`	There is a gentle ping and the Hai looks at a device on his wrist. "And now it seems my ship is finished refueling and I too must be about my business. Safe travels captain."`

Reviewer 2: ```suggestion
			`	There is a gentle ping, and the Hai looks at a device on his wrist. "And now it seems my ship is finished refueling, and I too must be about my business. Safe travels, Captain."`
```
